src/nvim/screen.c

```diff
@@ -3838,7 +3838,7 @@ static int win_line(win_T *wp, linenr_T lnum, int startrow, int endrow, bool noc
         if (n != 0) {
           /* At the window boundary, highlight the last character
            * instead (better than nothing). */
-          off += n;
+          off += (unsigned)n;
```

Comment:
This can cast a negative value to `unsigned`

Reply:
changing `off` to `int` then.